include/arch/arc/v2/mpu/arc_core_mpu.h

```diff
@@ -30,6 +30,7 @@ extern "C" {
 /* Thread Stack Region Intent Type */
 #define THREAD_STACK_REGION 0x1
 #define THREAD_STACK_GUARD_REGION 0x2
+#define THREAD_DOMAIN_PARTITION_REGION 0x3
```

Comment:
above comment mentions Stack regions only, please fix: and make it MPU Region Intent Type. Also, not sure if the THREAD_ prefix matches the domain partion region, memory domains are not thread specific like a stack.

Reply:
fixed